src/client/pages/Register.tsx

```diff
@@ -1,108 +1,84 @@
 import React from "react";
 
-import { Grid, Link, Typography } from "@material-ui/core";
+import { Button, Grid, Link, Typography } from "@material-ui/core";
 import TextField from "@material-ui/core/TextField";
 import Scaffold from "../components/Scaffold";
-import { View } from "../constants";
-import "./Register.css";
+import { getColor, View } from "../constants";
+import "./Authentication.css";
 
-export default function Register() {
-  return (
-    <Scaffold view={View.LOGIN}>
-      <div className="page-title">
-        <Typography variant="h1"> REGISTER </Typography>
-        <Typography className="register-decr">
-          Fill out all the information below to sign-up!
-          <Link href="/login" className="login-link" underline="hover">
+// interface Props {}
```

Comment:
Unnecessary comments

Reply:
those will be used once we actually start authenticating